src/index.html

```diff
@@ -10,7 +10,51 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
```

Comment:
```suggestion
    <form
```
Don't leave empty lines between parent and child elements